### ISSUE TITLE
AES_encrypt manpage is missing on openssl website in migration guide

### ISSUE THREAD

**Issue by vitalyk-radware** [openssl/openssl#23038]
https://www.openssl.org/docs/man3.0/man7/migration_guide.html 

says:
"Low-level encryption functions such as [AES_encrypt(3)](https://www.openssl.org/docs/man3.0/man3/AES_encrypt.html) and [AES_decrypt(3)](https://www.openssl.org/docs/man3.0/man3/AES_decrypt.html) have been informally discouraged from use for a long time"

but the link is broken

thank you!

**Comment by mspncp** [comment]
Indeed, the entire low-level AES API documenation is missing. It seems like it never existed. All I could find is third-party manual pages like this OpenBSD manual page for [AES_encrypt(3)](https://man.openbsd.org/AES_encrypt.3).

Given the fact that the low level AES API is deprecated, it's unlikely that some team member will step up to add it. If someone else takes on the task, that will certainly be welcome.

**Comment by jamuir** [comment]
`util/missingcrypto.txt` lists several functions that are missing documentation:

```
$ grep -e AES_encrypt -e AES_decrypt util/missingcrypto.txt
util/missingcrypto.txt:AES_decrypt(3)
util/missingcrypto.txt:AES_encrypt(3)
```

**Comment by bogomip2k-figma** [comment]
I'm happy to step in and help out here.  Can I literally just port over these other man pages from OpenBSD?

**Comment by t8m** [comment]
> I'm happy to step in and help out here. Can I literally just port over these other man pages from OpenBSD?

No, you cannot. All contributions to the OpenSSL codebase require signed CLA of all the authors - you would have to get all the contributors who wrote these man pages to sign the OpenSSL CLA.

**Comment by bogomip2k-figma** [comment]
Ah; that's probably why no one is tackling this.

**Comment by t8m** [comment]
Please note we are looking for community help implementing this issue. Please comment if you are interested in fixing the bug or if you have reasons why this bug is important enough so we should consider implementing it with higher priority than other requests. The issue will be closed after 2 months otherwise.

**Comment by t8m** [comment]
This issue was closed because it was marked with the help wanted label and there was no interest in implementing this issue or any other activity leading to implementing or re-prioritizing it for more than 2 months.
